python/core/auto_generated/qgsvectorlayer.sip.in

```diff
@@ -1216,7 +1216,7 @@ Deletes the selected features
 :return: ``True`` in case of success and ``False`` otherwise
 %End
 
-    QgsGeometry::OperationResult addRing( const QVector<QgsPointXY> &ring, QgsFeatureId *featureId = 0 );
+    QgsGeometry::OperationResult addRing( const QgsPointSequence &ring, QgsFeatureId *featureId = 0 );
```

Comment:
Again, these are all API breaks, and will require deprecation of the existing + add a new method